packages/node/src/transports/setup.ts

```diff
@@ -1,52 +0,0 @@
-import { getEnvelopeEndpointWithUrlEncodedAuth, initAPIDetails, NewTransport, NoopTransport } from '@sentry/core';
-import { Transport, TransportOptions } from '@sentry/types';
-import { makeDsn } from '@sentry/utils';
-
-import { NodeOptions } from '../types';
-import { HTTPSTransport, HTTPTransport, makeNodeTransport } from '.';
-
-/**
- * Sets up Node transport based on the passed `options`.
- *
- * @returns an object currently still containing both, the old `Transport` and
- * `NewTransport` which will eventually replace `Transport`. Once this is replaced,
- * this function will return a ready to use `NewTransport`.
- */
-// TODO(v7): Adjust return value when NewTransport is the default
-export function setupNodeTransport(options: NodeOptions): { transport: Transport; newTransport?: NewTransport } {
-  if (!options.dsn) {
-    // We return the noop transport here in case there is no Dsn.
-    return { transport: new NoopTransport() };
-  }
-
-  const dsn = makeDsn(options.dsn);
-
-  const transportOptions: TransportOptions = {
-    ...options.transportOptions,
-    ...(options.httpProxy && { httpProxy: options.httpProxy }),
-    ...(options.httpsProxy && { httpsProxy: options.httpsProxy }),
-    ...(options.caCerts && { caCerts: options.caCerts }),
```

Comment:
I'm wondering if we should make a breaking change that removes these fields as top level properties - and instead expect them to be passed in as transport options.